sqlalchemy_teradata/dialect.py

```diff
@@ -144,10 +156,10 @@ def _get_column_info(self, row):
                   2: 'UNICODE',
                   3: 'KANJISJIS',
                   4: 'GRAPHIC'}
-
+                                #Handle None characterset
         typ = self._resolve_type(row['columntype'],\
```

Comment:
in my local working version , i had " row['columntype'] or 'cv' " for this argument.  This addressed the old version of teradata (that my project is on) before the ColumnQV approach became available, and defaulted to a varchar in that case.   Looks like you may have addressed that in alternative ways, but something to think about.

Reply:
This was my first version too. After the "help type", the only case I can think of where the datatype will remain unknown would be when a view column is hardcoded to NULL.
For this situation, "None" is mapped to sqltypes.NullType.
Slightly off topic, but I am thinking that NullType can be a way to default any (yet) un-handled data types, as it would still allow to select and would not fail the reflection of the whole object. Would be nice to throw a warning whenever this is used though.